pkg/service/link.go

```diff
@@ -124,6 +124,49 @@ func pushLinksWithoutOperator(client kclient.ClientInterface, u unstructured.Uns
 	return nil
 }
 
+// DeleteLinkWithoutOperator deletes the ServiceBinding secret when Service Binding Operator is not installed
+func DeleteLinkWithoutOperator(kubeClient kclient.ClientInterface, secretToRemove unstructured.Unstructured, deployment *appsv1.Deployment) error {
+	currentNamespace := kubeClient.GetCurrentNamespace()
```

Comment:
The logic of this function has been ported from https://github.com/redhat-developer/odo/blob/224069cd68f4a03f4b5b17009fb4ca42ef8c5d89/pkg/service/link.go.